ui/src/actions/search.js

```diff
@@ -116,8 +120,18 @@ export function searchQueryUpdate(namespace, query) {
 
     const nextState = getState();
 
-    const helper = new SearchHelper(namespace, prevState, nextState);
-    searchConfig[namespace].onQueryChange(helper, dispatch);
+    const helper = new SearchHelper(
+      namespace,
+      prevState,
+      nextState,
+      dispatch,
+      dueToNavigationToSearchPage
+    );
+    searchConfig[namespace].onQueryChange(
+      helper,
+      dispatch,
+      dueToNavigationToSearchPage
```

Comment:
how come we didn't need this before for the synclocation? I didn't understand what change broke this

Reply:
https://github.com/inspirehep/inspirehep/blob/e233cb503fdf9e205aaa2c43197b59c96a681b99/ui/src/actions/search.js#L53-L64

That one.